pkg/katautils/utils.go

```diff
@@ -43,7 +45,9 @@ func IsEphemeralStorage(path string) bool {
 	if len(splitSourceSlice) > 1 {
 		storageType := splitSourceSlice[len(splitSourceSlice)-2]
 		if storageType == k8sEmptyDir {
-			return true
+			if _, fsType, _ := vc.GetDevicePathAndFsType(path); fsType == "tmpfs" {
```

Comment:
@amshinde under what other conditions is IsEphemeralStorage called. Only for volumes?

Reply:
@mcastelino Thats right.